Fix log trace: "No drm_driver.set_busid() implementation provided by nv_drm_driver [nvidia]."

Signed-off-by: Thomas Backlund <tmb@mageia.org>

diff -Nurp NVIDIA-Linux-x86_64-304.125.orig/kernel/nv-drm.c NVIDIA-Linux-x86_64-304.125/kernel/nv-drm.c
--- NVIDIA-Linux-x86_64-304.125.orig/kernel/nv-drm.c	2014-12-02 05:58:09.000000000 +0200
+++ NVIDIA-Linux-x86_64-304.125/kernel/nv-drm.c	2015-01-13 19:18:00.159015256 +0200
@@ -75,6 +75,9 @@ static struct drm_driver nv_drm_driver =
     .load = nv_drm_load,
     .unload = nv_drm_unload,
     .fops = &nv_drm_fops,
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 18, 0)
+    .set_busid = drm_pci_set_busid,
+#endif
 
     .name = "nvidia-drm",
     .desc = "NVIDIA DRM driver",
